### PR TITLE
www.yodobashi.com

### LINKED ISSUE #122105: www.yodobashi.com
### Issue URL (Social Widget)
 [https://www.yodobashi.com/product/100000001005418449/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.yodobashi.com%2Fproduct%2F100000001005418449%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #122105

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -2197,6 +2197,7 @@ androeed.ru##a[onclick*="#open_share_links"]
 thebridgechronicle.com,greaterkashmir.com,street-beat.ru##div[class^="social-share"]
 comss.ru##span[data-target="share-review"]
 yodobashi.com,utaten.com##.snsNav
+yodobashi.com##.pDetailCaption:has( > div.pDetailSNS)
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:2200]:
```
yodobashi.com#?#.pDetailCaption:has(> div.pDetailSNS)
```
Please move it to `extended_css` file

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt]: ok, done

File: SocialFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -7,6 +7,7 @@
 !
 academics.de##strong:contains(/^Diesen Job teilen$/)
 mobile-review.com#?#p:has(> strong:contains(Поделиться))
+yodobashi.com##.pDetailCaption:has( > div.pDetailSNS)
```
Comment (zloyden) [inline SocialFilter/sections/css_extended.txt:10]: You should add this `yodobashi.com#?#.pDetailCaption:has(> div.pDetailSNS)`.
Your rule has some mistakes.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: SocialFilter/sections/css_extended.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt]: sorry, and thank you for fixed my bad rule.

Comment (zloyden) [reply SocialFilter/sections/css_extended.txt]: You are welcome :)

**Pull request merged**